cleanlab_studio/studio/studio.py

```diff
@@ -104,7 +104,7 @@ def download_cleanlab_columns(
 
     def apply_corrections(self, cleanset_id: str, dataset: Any, keep_excluded: bool = False) -> Any:
         """
-        Applies corrections from a Cleanlab Studio cleanset to your dataset. Corrections can be made by viewing your project in the Cleanlab Studio webapp.
+        Applies corrections from a Cleanlab Studio cleanset to your dataset. Corrections can be made by viewing your project in the Cleanlab Studio webapp (see [Cleanlab Studio web quickstart](../../guide/quickstart/web#review-the-errors)).
```

Comment:
It seems that absolute links also work, `/guide/quickstart/web#review-the-errors`, should we use those instead?